src/components/Card/index.css

```diff
@@ -38,23 +38,34 @@
   max-width: 760px;
 }
 
-/* Course Card */
-.card-course .card-play-btn {
-  top: 50%;
+/* Card Play Button */
+.card-play-btn {
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.17);
+  left: 50%;
+  margin-left: -34px;
```

Comment:
Why do we have hard-coded values like this? Would be good to use flexible / responsive values + flexbox wherever possible.